### PR TITLE
1761-cookie-content-translation 

### PR DESCRIPTION
**Pull request by paulmsmith**
## Proposed changes

[Ticket here](https://trello.com/c/P6PLOCo1/1761-abstract-content-from-cookie-pages-in-funding-frontend)

### CODE REVIEW

File: app/views/help/cookie_details.html.erb [hunk 1]
```diff
@@ -2,80 +2,94 @@
...
+    <th scope="col" class="govuk-table__header"><%= t('cookie_details.table.th.purpose') %></th>
+    <th scope="col" class="govuk-table__header"><%= t('cookie_details.table.th.expires') %></th>
   </tr>
   </thead>
   <tbody class="govuk-table__body">
   <tr class="govuk-table__row">
     <td class="govuk-table__cell">_funding_frontend_ruby_session</th>
-    <td class="govuk-table__cell">Stores encrypted session data</td>
-    <td class="govuk-table__cell">When you close your browser</td>
+    <td class="govuk-table__cell"><%= t('cookie_details.table.td.stores_encrypted_session_data') %></td>
```
Comment (ptrelease) [inline app/views/help/cookie_details.html.erb:61]: PT: Run and check - looks like the translation lives under text, rather than td.

Comment (ptrelease) [reply app/views/help/cookie_details.html.erb:61]: Looks good.  Its repeated for td and text

**Review by ptrelease — APPROVED** [review]

File: app/views/help/cookie_details.html.erb (hunk 1 above)
Comment (stuartmccoll) [reply app/views/help/cookie_details.html.erb:61]: It looks that way to me too.

File: app/views/help/cookie_details.html.erb [hunk 2]
```diff
@@ -84,23 +98,23 @@
...
+  <td class="govuk-table__cell"><%= t('cookie_details.table.td.when_you_close_your_browser') %></td>
 </tr>
 </tbody>
 </table>
 
-<h2 class="govuk-heading-m">Change your settings</h2>
-<p class="govuk-body">You can <%=link_to 'change which cookies you’re happy for us to use', help_cookies_path %>.</p>
\ No newline at end of file
+<h2 class="govuk-heading-m"><%= t('cookie_details.headings.change_your_settings') %></h2>
+<p class="govuk-body">You can <%= link_to t('cookie_details.links.change_which_cookies_youre_happy_for_us_to_use'), help_cookies_path %>.</p>
```
Comment (stuartmccoll) [inline app/views/help/cookie_details.html.erb:120]: Missed the 'You can' at the start of this `p` element.

File: config/locales/help/cookie_details/en.yml [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+      3: "Hotjar stores anonymised information about:"
+    links:
+      change_which_cookies_youre_happy_for_us_to_use: "change which cookies you’re happy for us to use"
+    li:
+      1: "the pages you visit on Apply for Funding and how long you spend on each page,"
+      2: "how you navigate around the pages you look at on Apply for Funding,"
+      3: "what you click on while you’re using the service,"
+      4: "how you fill in the form in Apply for Funding,"
+      5: "your keystrokes, and any address or financial data you input, are not recorded or stored."
+    error: "Error:"
```
Comment (stuartmccoll) [inline config/locales/help/cookie_details/en.yml:39]: Is this one being used? We've got some generic abstractions in the top-level `en.yml` where you'll find things like error, warning, etc.

**Review by stuartmccoll — CHANGES_REQUESTED** [review]
Have left a couple of comments.

**Comment by paulmsmith** [comment]
Good spots. Resolved those comments. @ptrelease @stuartmccoll

**Review by stuartmccoll — APPROVED** [review]

**Pull request merged**